### PR TITLE
fix: syntax error in readme example

### PR DESCRIPTION
**Pull request by mingrammer**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -179,7 +181,7 @@ func main() {
     //            Options include MaxRetry, Queue, Timeout, Deadline, Unique etc.
     // ----------------------------------------------------------------------------
 
-    c.SetDefaultOptions(tasks.ImageProcessing, asynq.MaxRetry(10), asynq.Timeout(3*time.Minute))
+    c.SetDefaultOptions(tasks.ImageProcessor, asynq.MaxRetry(10), asynq.Timeout(3*time.Minute))
```
Comment (hibiken) [inline README.md:184]: the first argument should be a task type, so it should be `tasks.TypeImageResize`

**Review by hibiken — COMMENTED** [review]
Thanks for spotting this error!
I'll approve this PR once the comment is resolved 👍

File: README.md (hunk 1 above)
Comment (mingrammer) [reply README.md]: I fixed it!

**Comment by hibiken** [comment]
Great! One last ask: Would you mind squashing these two commits to one?

**Comment by mingrammer** [comment]
@hibiken You can squash the commits when merging the PR :) You can find it by open the dropbox menu on the arrow button next to the merge button.)

**Pull request merged**